src/scenes/mainScene.ts

```diff
@@ -86,6 +89,54 @@ export class MainScene extends Scene {
         this.playerTurnInfo.setText(this.player.turn.toString());
         this.updateBuildFactoryButton();
         this.updateCityInfos();
+        this.updateVisibility();
```

Comment:
who's visibility is updated? The + - buttons?

Reply:
Name has been changed. New pull request will follow immediately. I was planning to extend the method to handle more GameObjects but it does not seem necessary at the moment.